graph/telemetry/istio/appender/ambient_test.go

```diff
@@ -192,50 +134,13 @@ func TestRemoveWaypoint(t *testing.T) {
 	assert.False(found)
 }
 
-func TestIsWaypoint(t *testing.T) {
```

Comment:
I wonder if it would be possible to add some test coverage for the code that determines if is a waypoint (https://github.com/kiali/kiali/pull/7742/files#diff-611af3df8bfed322d3a0a785476482260509aa84540c2f8d70230b065ddf4820R1015)

Reply:
This is tested in an indirect way, in the new ambient test case in api_test.go.  That test will fail if for some reason we can't detect the waypoint workloads.